packages/search/lib/commands/ALIASADD.ts

```diff
@@ -1,3 +1,5 @@
+export const FIRST_KEY_INDEX = 2;
```

Comment:
Actually `2` is incorrect as well, this should be executed on all shards (same for all alias commands)